src/PackageHelper.cs

```diff
@@ -215,17 +265,112 @@ public HttpClient GetClientBase(string url, string apiKey, int memberId, int pro
             var headerToken = HMACAuthentication.GenerateAuthorizationHeader(signature, nonce, timestamp);
 
             client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", headerToken);
-            client.DefaultRequestHeaders.Add(AuthConstants.MemberIdHeader, memberId.ToInvariantString());
-            client.DefaultRequestHeaders.Add(AuthConstants.ProjectIdHeader, projectId.ToInvariantString());
-            
+            client.DefaultRequestHeaders.Add(AuthConstants.MemberIdHeader, memberId.ToString());
+            client.DefaultRequestHeaders.Add(AuthConstants.ProjectIdHeader, projectId.ToString());
```

Comment:
Although integers don't have formatting differences besides the negative sign (which isn't valid for these IDs), it's probably best to keep the invariant culture format provider:

```suggestion
            client.DefaultRequestHeaders.Add(AuthConstants.MemberIdHeader, memberId.ToString(CultureInfo.InvariantCulture));
            client.DefaultRequestHeaders.Add(AuthConstants.ProjectIdHeader, projectId.ToString(CultureInfo.InvariantCulture));
```